uav_names: [
    uav1, uav2, uav3,
  ]